tests/visual/percy-storybook/DashTable.percy.tsx

```diff
@@ -200,38 +200,34 @@ const sparseData = (() => {
     ));
 })();
 
-const hiddenColumns = R.addIndex(R.map)((column, index) =>
-    R.mergeAll([
-        {},
-        column,
-        { hidden: index % 2 === 0 }
-    ]),
-    columnsA2J
-);
+const hiddenColumns = columnsA2J.map(c => c.id).filter((_, index) => index % 2 === 0);
 
 storiesOf('DashTable/Hidden Columns', module)
     .add('hides', () => (<DataTable
         setProps={setProps}
         id='table'
         data={dataA2J}
-        columns={hiddenColumns}
+        columns={columnsA2J}
+        hidden_columns={hiddenColumns}
```

Comment:
These changes were overlooked in the 4.0 work -- `hidden` wasn't supported anymore. Now changed to use `hidden_columns` instead